src/main/java/org/mitre/hapifhir/MedMorphToCIBMTR.java

```diff
@@ -24,35 +32,38 @@
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.impl.client.HttpClients;
 import org.apache.http.util.EntityUtils;
-import java.util.stream.Collectors;
 
 public class MedMorphToCIBMTR {
   private String cibmtrUrl;
   private String ccn;
+  private String authToken;
```

Comment:
We should clarify how this class is intended to be used. Do we intend (a) just one instance of MedMorphToCIBMTR where every request invokes `convert` on that instance, or (b) is there one instance of MedMorphToCIBMTR per request? If (a), then we don't want the CCN and auth token to be stored as state - they should be passed to the various functions as needed. This is fine if (b) though. I was originally assuming (a) but based on what's on the `cibmtr` branch on medmorph-fhir-server it seems like we're going with (b). If that's the case can you add some comments saying that's the intended use?

Reply:
I think we actually do want (a) where we only have one instance of `MedMorphToCIBMTR` so I'll remove `ccn` and `authToken` as state and change how I instantiate `MedMorphToCIBMTR` on the `cibmtr` branch.